lib/email/sender.rb

```diff
@@ -430,9 +447,16 @@ def fix_parts_after_attachments!
       end
     end
 
+    # NOTE: In most cases this is not a problem, but if a header has
+    # doubled up the header[] method will return an array. So we always
+    # get the last value of the array and assume that is the correct
+    # value.
+    #
+    # See https://www.rubydoc.info/github/mikel/mail/Mail/Header
```

Comment:
Should this comment site directly above the code that it is describing?

Reply:
Yeah I suppose, will move it down slightly